qase-api/src/main/java/io/qase/client/services/Attachments.java

```diff
@@ -0,0 +1,97 @@
+package io.qase.client.services;
```

Comment:
io.qase.client package contains only classes generated from openapi specification. Move your classes to another package